banzai/stages.py

```diff
@@ -160,10 +160,11 @@ def do_stage(self, images):
 
             if master_calibration_filename is None:
                 self.on_missing_master_calibration(logging_tags)
-
-            master_calibration_image = Image(self.pipeline_context,
+            else:
```

Comment:
I think this will break things down the road. If there is no master calibration it makes a default image object that can be used later down the line. This would cause a bunch of extra exceptions.

Reply:
Reverted